.github/workflows/run-tests.yml

```diff
@@ -67,7 +73,24 @@ jobs:
             with-3d: OFF
             with-quick: OFF
             patch-qt-3d: false
+            with-grass7: OFF
+            with-grass8: OFF
             LD_PRELOAD: ''
+            experimental: false
+
+          - distro-version: '36'
```

Comment:
How would you feel about adapting the existing qt 6 test instead? That'd save adding another lengthy job to the CI...

Reply:
reasoning for duplicating is GRASS8 needs `WERROR=false`, cause there are some nasty warnings.
If you are ready to accept `WERROR=false` for QT6 then let's go and disable it...
see details at https://cdash.orfeo-toolbox.org/index.php?project=QGIS&filtercount=1&showfilters=1&field1=buildname/string&compare1=63&value1=fix-libgrass_imagery_863cd4f6130733d14d03e02b2fc11e3e21f52b64&date=2022-03-17

Reply:
Ok, that's a fair point! But I'm still reticent to add the overhead of another lengthy CI job (with the associated cost in time/energy that entails) just for the undermaintained grass provider. (Unless this change is a prequel to some maintenance work on that provider, which would be VERY welcome 😁!)

Reply:
true it's under maintained. that's why I suggest to address it by temporarily adding this CI job to facilitate maintenance work on the grass provider. I look at it as a prequel. (after grass got some love this job should replace the old one.)

Reply:
after #47937 and #47961 there are no `Werror`s left... so let's go with a bumped 34->36 for qt6 instead